### PR TITLE
9463 spell mistakes 2019 nov

### PR DESCRIPTION
**Pull request by raghavgai**
Fixed few spelling mistakes while going through code.

### CODE REVIEW

**Comment by bryevdv** [comment]
@raghavgai Thanks for the PR. However the change in `test_compiler.py` will need to be reverted, as that error is actually intentional&mdash;the test is to maintain that the error message is informative in the expected way (so no that it is "fixed" it is actually failing).

File: tests/unit/bokeh/util/test_compiler.py [hunk 1]
```diff
@@ -45,12 +45,12 @@ def test_nodejs_compile_javascript():
...
 
-    assert buc.nodejs_compile("""function f(a, b) { eturn a + b; };""", "javascript", "some.js") == \
+    assert buc.nodejs_compile("""function f(a, b) { return a + b; };""", "javascript", "some.js") == \
         dict(error=
             '\x1b[96msome.js\x1b[0m:\x1b[93m1\x1b[0m:\x1b[93m26\x1b[0m - '
             "\x1b[91merror\x1b[0m\x1b[90m TS1005: \x1b[0m';' expected.\n"
             '\n'
-            '\x1b[7m1\x1b[0m function f(a, b) { eturn a + b; };\n'
+            '\x1b[7m1\x1b[0m function f(a, b) { return a + b; };\n'
             '\x1b[7m \x1b[0m \x1b[91m                         ~\x1b[0m\n')
```
Comment (bryevdv) [inline tests/unit/bokeh/util/test_compiler.py]: The errors in this test are intentional (the test is testing an error reporting capability) so they will need to be reverted.

**Comment by bryevdv** [comment]
In the interest of clearing this I have just made the necessary edits in the GH web UI

**Comment by raghavgai** [comment]
Thanks! @bryevdv

**Pull request merged**